birdhouse/scripts/read-configs.include.sh

```diff
@@ -55,7 +66,7 @@ discover_compose_dir() {
             # Case of sub-subdir of sibling checkout at same level as birdhouse-deploy.
             COMPOSE_DIR="$(realpath "../../../birdhouse-deploy/birdhouse")"
         fi
-        echo "$COMPOSE_DIR"
+        echo "${MSG_INFO}Resolved docker-compose directory: [${COMPOSE_DIR}]"
```

Comment:
Oh maybe avoid this logging here.  I think some script might do `get_compose_dir=$(discover_compose_dir)` and this will break it.  I can't remember where though, so I might hallucinate.

Reply:
Wouldn't `get_compose_dir=$(discover_compose_dir)` return an empty string? 
The value is set with `export COMPOSE_DIR` just below, but it is never echo'ed by this function.